### ISSUE TITLE
Game crashes whenever I try to join a global room/any levels

### ISSUE THREAD

**Issue by Geming400** [dankmeme01/globed2#1]
I tried to join a global room, but whenever I did this, my game crashed.
Here is the crash log:
```
2024-02-07T16:13:10+0100
Whoopsies! An unhandled exception has occurred.

== Geode Information ==
Loader Version: v2.0.0-beta.15
Loader Commit: d35b2de
Bindings Commit: 253a854
Installed mods: 22
Problems: 0

== Exception Information ==
Faulty Module: D:\SteamLibrary\steamapps\common\Geometry Dash\GeometryDash.exe
Faulty Mod: <Unknown>
Exception Code: c0000005 (EXCEPTION_ACCESS_VIOLATION)
Exception Flags: 0
Exception Address: 41380000 (41380000)
Number Parameters: 2

== Stack Trace ==
 - 41380000
 - D:\SteamLibrary\steamapps\common\Geometry Dash\geode\unzipped\dankmeme.globed2\dankmeme.globed2.dll + 80268
 - D:\SteamLibrary\steamapps\common\Geometry Dash\Geode.dll + 2da7f (geode::EventFilter<geode::ModInstallEvent>::setListener + 9967)
 - 00DC1B17
 - 1E2A23A9
 - 1E2A21CB
 - D:\SteamLibrary\steamapps\common\Geometry Dash\GeometryDash.exe + 3c12d8
 - D:\SteamLibrary\steamapps\common\Geometry Dash\Geode.dll + c64b6 (geode::utils::web::AsyncWebRequest::userAgent + 12406)
 - D:\SteamLibrary\steamapps\common\Geometry Dash\GeometryDash.exe + 3c267e
 - C:\Windows\System32\KERNEL32.DLL + 1fcc9 (BaseThreadInitThunk + 25)
 - C:\Windows\SYSTEM32\ntdll.dll + 67c6e (RtlGetAppContainerNamedObjectPath + 286)
 - C:\Windows\SYSTEM32\ntdll.dll + 67c3e (RtlGetAppContainerNamedObjectPath + 238)

== Register States ==
EAX: 2d3884b8
EBX: 2cf54828
ECX: 2cf518b0
EDX: 0224cb74
EBP: 00d6f81c
ESP: 00d6f7a4
EDI: 2cf518b0
ESI: 049d6928
EIP: 41380000

== Installed Mods ==
x | [v8.0.0-alpha.9] absolllute.megahack
x | [v1.3.4] alphalaneous.integratedloquibot
x | [v1.1.2] alphalaneous.projectedstars
x | [v1.1.2] capeling.demons_in_garage_2.2
x | [v4.0.0-alpha.12] cvolton.betterinfo
x | [v1.0.0] dankmeme.globed2
x | [v2.4.0] fleym.nongd
x | [v1.2.14] gdutilsdevs.gdutils
x | [v1.4.0] geode.custom-keybinds
x | [v2.0.0-beta.15] geode.loader
x | [v1.3.0] geode.node-ids
x | [v1.0.4] hjfod.gdshare
x | [v1.0.1] hjfod.gmd-api
x | [v1.0.1] itzkiba.elegant_rankings
  | [v1.1.0] itzkiba.grandpa_demon
x | [v1.1.3] mat.circle-tool
x | [v1.1.0] mat.scroll-zoom
x | [v1.6.0-beta.2] spaghettdev.gd-roulette
x | [v1.0.0] techstudent10.hide_locks
x | [v1.1.0] thesillydoggo.better_startpos_menu
  | [v2.1.0] TheSillyDoggo.GradientPages
x | [v1.2.0] thesillydoggo.moretags
```
Here is also a video:

https://github.com/dankmeme01/globed2/assets/101340400/7d47fd9b-14da-4a7a-b877-78332b605eab

---------------------

Going into any levels just also crash the game

https://github.com/dankmeme01/globed2/assets/101340400/b6956813-f3b8-4847-b30d-8a63443d17bb
https://github.com/dankmeme01/globed2/assets/101340400/f93fdb14-4c81-47f2-8464-b8a46aeb12ab

The crash log for this is *(normally)* the same:

```
(bloodbath = the level with some players in it)
2024-02-07T16:27:25+0100
Whoopsies! An unhandled exception has occurred.

== Geode Information ==
Loader Version: v2.0.0-beta.15
Loader Commit: d35b2de
Bindings Commit: 253a854
Installed mods: 22
Problems: 0

== Exception Information ==
Faulty Module: D:\SteamLibrary\steamapps\common\Geometry Dash\GeometryDash.exe
Faulty Mod: <Unknown>
Exception Code: c0000005 (EXCEPTION_ACCESS_VIOLATION)
Exception Flags: 0
Exception Address: 00000000 (00000000)
Number Parameters: 2

== Stack Trace ==
 - 00000000
 - D:\SteamLibrary\steamapps\common\Geometry Dash\geode\unzipped\dankmeme.globed2\dankmeme.globed2.dll + 80268
 - D:\SteamLibrary\steamapps\common\Geometry Dash\Geode.dll + 2da7f (geode::EventFilter<geode::ModInstallEvent>::setListener + 9967)
 - 00BF1B17
 - 18AE2389
 - 18AE21AB
 - D:\SteamLibrary\steamapps\common\Geometry Dash\GeometryDash.exe + 3c12d8
 - D:\SteamLibrary\steamapps\common\Geometry Dash\Geode.dll + c64b6 (geode::utils::web::AsyncWebRequest::userAgent + 12406)
 - D:\SteamLibrary\steamapps\common\Geometry Dash\GeometryDash.exe + 3c267e
 - C:\Windows\System32\KERNEL32.DLL + 1fcc9 (BaseThreadInitThunk + 25)
 - C:\Windows\SYSTEM32\ntdll.dll + 67c6e (RtlGetAppContainerNamedObjectPath + 286)
 - C:\Windows\SYSTEM32\ntdll.dll + 67c3e (RtlGetAppContainerNamedObjectPath + 238)

== Register States ==
EAX: 2c6af380
EBX: 2c490440
ECX: 2c48d4c8
EDX: 00000001
EBP: 00b9f564
ESP: 00b9f4ec
EDI: 2c48d4c8
ESI: 04e72810
EIP: 00000000

== Installed Mods ==
x | [v8.0.0-alpha.9] absolllute.megahack
x | [v1.3.4] alphalaneous.integratedloquibot
x | [v1.1.2] alphalaneous.projectedstars
x | [v1.1.2] capeling.demons_in_garage_2.2
x | [v4.0.0-alpha.12] cvolton.betterinfo
x | [v1.0.0] dankmeme.globed2
x | [v2.4.0] fleym.nongd
x | [v1.2.14] gdutilsdevs.gdutils
x | [v1.4.0] geode.custom-keybinds
x | [v2.0.0-beta.15] geode.loader
x | [v1.3.0] geode.node-ids
x | [v1.0.4] hjfod.gdshare
x | [v1.0.1] hjfod.gmd-api
x | [v1.0.1] itzkiba.elegant_rankings
  | [v1.1.0] itzkiba.grandpa_demon
x | [v1.1.3] mat.circle-tool
x | [v1.1.0] mat.scroll-zoom
x | [v1.6.0-beta.2] spaghettdev.gd-roulette
x | [v1.0.0] techstudent10.hide_locks
x | [v1.1.0] thesillydoggo.better_startpos_menu
  | [v2.1.0] TheSillyDoggo.GradientPages
x | [v1.2.0] thesillydoggo.moretags
```

```
(The recent tab level crash log)
2024-02-07T16:30:36+0100
Whoopsies! An unhandled exception has occurred.

== Geode Information ==
Loader Version: v2.0.0-beta.15
Loader Commit: d35b2de
Bindings Commit: 253a854
Installed mods: 22
Problems: 0

== Exception Information ==
Faulty Module: D:\SteamLibrary\steamapps\common\Geometry Dash\GeometryDash.exe
Faulty Mod: <Unknown>
Exception Code: c0000005 (EXCEPTION_ACCESS_VIOLATION)
Exception Flags: 0
Exception Address: 00000000 (00000000)
Number Parameters: 2

== Stack Trace ==
 - 00000000
 - D:\SteamLibrary\steamapps\common\Geometry Dash\geode\unzipped\dankmeme.globed2\dankmeme.globed2.dll + 80268
 - D:\SteamLibrary\steamapps\common\Geometry Dash\Geode.dll + 2da7f (geode::EventFilter<geode::ModInstallEvent>::setListener + 9967)
 - 01321B17
 - 1D8A2389
 - 1D8A21AB
 - D:\SteamLibrary\steamapps\common\Geometry Dash\GeometryDash.exe + 3c12d8
 - D:\SteamLibrary\steamapps\common\Geometry Dash\Geode.dll + c64b6 (geode::utils::web::AsyncWebRequest::userAgent + 12406)
 - D:\SteamLibrary\steamapps\common\Geometry Dash\GeometryDash.exe + 3c267e
 - C:\Windows\System32\KERNEL32.DLL + 1fcc9 (BaseThreadInitThunk + 25)
 - C:\Windows\SYSTEM32\ntdll.dll + 67c6e (RtlGetAppContainerNamedObjectPath + 286)
 - C:\Windows\SYSTEM32\ntdll.dll + 67c3e (RtlGetAppContainerNamedObjectPath + 238)

== Register States ==
EAX: 2c5ac9e0
EBX: 2c3c6628
ECX: 2c3c36b0
EDX: 0794a00c
EBP: 012ffa4c
ESP: 012ff9d4
EDI: 2c3c36b0
ESI: 053428e8
EIP: 00000000

== Installed Mods ==
x | [v8.0.0-alpha.9] absolllute.megahack
x | [v1.3.4] alphalaneous.integratedloquibot
x | [v1.1.2] alphalaneous.projectedstars
x | [v1.1.2] capeling.demons_in_garage_2.2
x | [v4.0.0-alpha.12] cvolton.betterinfo
x | [v1.0.0] dankmeme.globed2
x | [v2.4.0] fleym.nongd
x | [v1.2.14] gdutilsdevs.gdutils
x | [v1.4.0] geode.custom-keybinds
x | [v2.0.0-beta.15] geode.loader
x | [v1.3.0] geode.node-ids
x | [v1.0.4] hjfod.gdshare
x | [v1.0.1] hjfod.gmd-api
x | [v1.0.1] itzkiba.elegant_rankings
  | [v1.1.0] itzkiba.grandpa_demon
x | [v1.1.3] mat.circle-tool
x | [v1.1.0] mat.scroll-zoom
x | [v1.6.0-beta.2] spaghettdev.gd-roulette
x | [v1.0.0] techstudent10.hide_locks
x | [v1.1.0] thesillydoggo.better_startpos_menu
  | [v2.1.0] TheSillyDoggo.GradientPages
x | [v1.2.0] thesillydoggo.moretags
```

**Comment by Alyoshadash** [comment]
What should I do if my GD just crashes? I even deleted all mods (except custom keybinding) well, it still doesn't help. How can I fix this? (I can send crashlogs of each error)

**Comment by dankmeme01** [comment]
will be fixed in release v1.0.2 (https://github.com/dankmeme01/globed2/commit/58a631f927a5df5b1273a0df12bf496f12fdbede)

**Comment by Geming400** [comment]
Ok thk! Have fun continuing this :)
